doc/source/examples/bo_with_warpx.rst

```diff
@@ -0,0 +1,75 @@
+.. _bo-with-warpx:
+
+Optimization with WarpX
+=======================
+
+
+Description
+~~~~~~~~~~~
+
+This examples shows how to perform a Bayesian optimization of a laser-plasma
+accelerator (LPA) using WarpX simulations.
+
+The LPA to be optimized is based on
```

Comment:
This sentence seems incomplete.

Reply:
Yep, that was a stub I meant to delete.  It's gone now